core/src/services/s3/backend.rs

```diff
@@ -54,7 +54,8 @@ static ENDPOINT_TEMPLATES: Lazy<HashMap<&'static str, &'static str>> = Lazy::new
     m
 });
 
-/// Aws S3 and compatible services (including minio, digitalocean space, Tencent Cloud Object Storage(COS) and so on) support
+/// Aws S3 and compatible services (including minio, digitalocean space, Tencent Cloud Object Storage(COS) and so on) support.
+/// For more information about s3-compatible services, refer to [Compatible Services](https://opendal.apache.org/docs/rust/opendal/services/struct.S3.html#compatible-services).
```

Comment:
We should not hardcode the links here because we could serve them from different host.

Use `[Compatible Services](#compatible-services)` instead.